lib/Basics/NumberOfCores.cpp

```diff
@@ -45,12 +47,29 @@ std::size_t numberOfCoresImpl() {
     n = 0;
   }
 
-  if (n > 0) {
-    return static_cast<std::size_t>(n);
+  if (n <= 0)
+#endif
+    {
+      n = std::thread::hardware_concurrency();
+    }
+#ifdef TRI_SC_NPROCESSORS_ONLN
+  try {
+    int64_t cfsPeriodUs;
+    int64_t cfsQuotaUs;
+    cfsPeriodUs = std::stoll(basics::FileUtils::slurp("/sys/fs/cgroup/cpu/cpu.cfs_period_us"));
+    cfsQuotaUs = std::stoll(basics::FileUtils::slurp("/sys/fs/cgroup/cpu/cpu.cfs_quota_us"));
+    if (cfsQuotaUs > 0 && cfsPeriodUs > 0) {
+      n = std::lround(double(cfsQuotaUs) / double(cfsPeriodUs));
+    }
```

Comment:
Can be simplified to just:
```suggestion
    int64_t cfsPeriodUs = std::stoll(basics::FileUtils::slurp("/sys/fs/cgroup/cpu/cpu.cfs_period_us"));
    int64_t cfsQuotaUs = std::stoll(basics::FileUtils::slurp("/sys/fs/cgroup/cpu/cpu.cfs_quota_us"));
    if (cfsQuotaUs > 0 && cfsPeriodUs > 0) {
      n = std::lround(double(cfsQuotaUs) / double(cfsPeriodUs));
    }
```
If `cfsQuotaUs` and `cfsPeriodUs` are supposed to be real values, why don't we use `std::stod` (apart from that it is locale-dependent)?